src/main/java/javax/validation/Validation.java

```diff
@@ -273,7 +275,7 @@ public ValidationProviderResolver getDefaultValidationProviderResolver() {
 			if ( validationProviders.size() == 0 ) {
 				String msg = "Unable to create a Configuration, because no Bean Validation provider could be found." +
```

Comment:
What difference does it make in the end? Constants are copied inline by the compiler.